test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -79,18 +83,26 @@ def clone(x):
 
         return tuple(clone(x) for x in inputs)
 
-    def _generate_reference_quantized_model(self, mod, inputs):
-        export_model = capture_pre_autograd_graph(
-            mod,
-            inputs,
-        )
-        quantizer = X86InductorQuantizer()
-        quantizer.set_global(xiq.get_default_x86_inductor_quantization_config())
-        prepare_model = prepare_pt2e(export_model, quantizer)
-        prepare_model(*inputs)
-        convert_model = convert_pt2e(prepare_model)
-        torch.ao.quantization.move_exported_model_to_eval(convert_model)
-        return convert_model
+    def _generate_reference_quantized_model(self, mod, inputs, is_qat=False):
```

Comment:
nit: let's call this qdq_quantized_model, to align with https://pytorch.org/tutorials/prototype/pt2e_quant_ptq_static.html#convert-the-calibrated-model-to-a-quantized-model

Reply:
Thanks for the comment and changed.